manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -216,16 +235,20 @@ public static function setProjectDir(string $projectDir): void
         self::$projectDir = realpath($projectDir) ?: $projectDir;
     }
 
-    public static function create(string $projectDir, bool $debug = false): self
+    public static function create(string $projectDir, string $env = null): self
     {
-        if (file_exists($projectDir.'/.env')) {
-            (new Dotenv(false))->load($projectDir.'/.env');
+        self::loadEnv($projectDir);
+
+        if (null === $env) {
+            $env = (string) ($_SERVER['APP_ENV'] ?? $_SERVER['SYMFONY_ENV'] ?? 'prod');
```

Comment:
`SYMFONY_ENV` is the legacy key from the pre-flex era. Do we really want to support it?